airflow/models/baseoperator.py

```diff
@@ -258,7 +259,7 @@ def __init__(
         priority_weight: int = 1,
         weight_rule: str = WeightRule.DOWNSTREAM,
         queue: str = configuration.conf.get('celery', 'default_queue'),
-        pool: Optional[str] = None,
+        pool: Optional[str] = Pool.DEFAULT_POOL_NAME,
```

Comment:
```suggestion
        pool: str = Pool.DEFAULT_POOL_NAME,
```

Not optional any more

Reply:
Fixed.